pkg/sessions/redis/redis_store.go

```diff
@@ -103,6 +101,20 @@ func (store *SessionStore) ClearAll(ctx context.Context, s *sessions.SessionStat
 	return nil
 }
 
+func saveTicketWithLock(ctx context.Context, store *SessionStore, keyName string, ticket string, exp time.Duration) error {
+	lock := store.Lock(ticket)
```

Comment:
Seems like we should still be using keyName for the lock argument here?

Reply:
yes, done the chane